### PR TITLE
feat: add time unit and amount for digest

### PR DESCRIPTION
**Pull request by davidsoderberg**

### CODE REVIEW

**Comment by davidsoderberg** [comment]
@ainouzgali  and @scopsy please help with copy for the form fields labels 🙏

File: libs/dal/src/repositories/job/job.entity.ts [hunk 1]
```diff
@@ -27,5 +27,7 @@ export class JobEntity {
   _templateId: string;
   digest?: {
     events?: any[];
+    amount?: number;
+    unit?: string;
```
Comment (scopsy) [inline libs/dal/src/repositories/job/job.entity.ts:31]: Could you elaborate on adding this two here? As opposed to fetching them from the step entity in runtime?

File: apps/web/src/pages/templates/workflow/WorkflowEditorPage.tsx [hunk 2]
```diff
@@ -138,6 +139,13 @@ const WorkflowEditorPage = ({
                     })}
                   </NavSection>
                 </When>
+                <When truthy={selectedChannel === ChannelTypeEnum.DIGEST}>
```
Comment (scopsy) [inline apps/web/src/pages/templates/workflow/WorkflowEditorPage.tsx:142]: I just love this When component 😃

File: apps/web/src/components/templates/TemplateFormProvider.tsx [hunk 3]
```diff
@@ -86,6 +123,8 @@ export const TemplateFormProvider = ({ children }) => {
     resolver: zodResolver(schema),
   });
 
+  console.log(methods.formState.errors);
```
Comment (scopsy) [inline apps/web/src/components/templates/TemplateFormProvider.tsx:126]: I think this one slipped here :)

**Review by scopsy — COMMENTED** [review]
This looks great @davidsoderberg ! Added a few questions and suggestions let me know what you think

File: apps/web/src/pages/templates/workflow/WorkflowEditorPage.tsx (hunk 2 above)
Comment (davidsoderberg) [reply apps/web/src/pages/templates/workflow/WorkflowEditorPage.tsx:142]: I have a more advanced in store for `When` we need it 😉

File: libs/dal/src/repositories/job/job.entity.ts (hunk 1 above)
Comment (davidsoderberg) [reply libs/dal/src/repositories/job/job.entity.ts:31]: `As opposed to fetching them from the step entity in runtime?`
Yes exactly 👍

**Review by ainouzgali — APPROVED** [review]

**Pull request merged**